test/test_gpu_transpose.cpp

```diff
@@ -1029,4 +1029,47 @@ TEST_F(NVFuserTest, FusionTransposeBankConflict8_CUDA) {
   TORCH_CHECK(bank_conflict_info.empty());
 }
 
+TEST_F(NVFuserTest, FusionTransposeBankConflict9_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto tv0 = makeConcreteTensor({32, 32, 2});
+  fusion.addInput(tv0);
+  auto tv1 = set(tv0);
+  auto tv2 = transpose(tv1, 0, 1);
+  auto tv3 = set(tv2);
+  fusion.addOutput(tv2);
+
+  tv1->setMemoryType(MemoryType::Shared);
+
+  tv1->merge(0);
+  tv1->merge(0);
+  tv1->split(0, 4);
+  tv1->split(0, 32);
+  tv1->axis(-1)->parallelize(ParallelType::Vectorize);
+  tv1->axis(-2)->parallelize(ParallelType::TIDx);
+
+  for (auto tv : {tv2, tv3}) {
```

Comment:
With https://github.com/NVIDIA/Fuser/pull/148, I can finally vectorize of the consumer of transpose, this is a good thing. However, I also observed that, if I vectorize the consumer of transpose illegally, we can still lowering to a kernel and running that kernel get misaligned address error. This might be wrong. I am expecting to see some failure on lowering validation.

Reply:
Didn't we disable some validations as they are not updated to handle some of the (legal) vectorizations done by the schedulers?

Reply:
I am not sure. I know that validation of vectorization is hard, and we disabled some validation. But I don't know if the validation needed here is the one disabled.